README.md

```diff
@@ -96,6 +96,61 @@ If you are interested in helping, please reach out!
 You can use GitHub or our [Discord server](https://discord.gg/pxrBmy4).
 
 
+## Android Integration
+
+The Android implementation requires the application's Activity to forward input events (and
+input devices) to the plugin. Below is an example of a MainActivity for a clean Flutter project
+that has implemented the required boilerplate code. For many projects it will be possible to simply
+duplicate this setup.
+
+```dart
+package [YOUR_PACKAGE_NAME]
+
+import android.hardware.input.InputManager
+import android.os.Handler
+import android.view.InputDevice
+import android.view.KeyEvent
+import android.view.MotionEvent
+import io.flutter.embedding.android.FlutterActivity
+import dev.markvideon.gamepads_android.GamepadsCompatibleActivity
+
+class MainActivity: FlutterActivity(), GamepadsCompatibleActivity {
+    var keyListener: ((KeyEvent) -> Boolean)? = null
+    var motionListener: ((MotionEvent) -> Boolean)? = null
+
+    override fun dispatchGenericMotionEvent(motionEvent: MotionEvent): Boolean {
+        motionListener?.invoke(motionEvent)
+        return true
+    }
+
+    override fun dispatchKeyEvent(keyEvent: KeyEvent): Boolean {
+        keyListener?.invoke(keyEvent)
+        return true
```

Comment:
I think this should actually be changed so that non-gamepad key events are still propagated to Flutter. 
i.e. https://github.com/flame-engine/gamepads/issues/7